### DISCUSSION TITLE
Cannot change the default font

### DISCUSSION THREAD

**Discussion by wsw70** [tailwindlabs/tailwindcss#12124] [Help] [answered]
The documentation is [very clear][1] on how to change the default font for the whole application:

> For convenience, **Preflight** sets the font family on the `html` element to match your configured sans font, so one way to change the default font for your project is to customize the sans key in your `fontFamily` configuration:

In an application I started from scratch, I did that and added to the main page (this is a Vite/Vue3 project that starts by default with an `App.vue` page and everything is routed from there)

```css
<style>
@import url('https://fonts.googleapis.com/css2?family=Nunito:ital,wght@0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;0,1000;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900;1,1000&display=swap');
</style>
```

Together with 

```js
/** @type {import('tailwindcss').Config} */
export default {
  content: [
    "./index.html",
    "./src/**/*.{vue,js,ts,jsx,tsx}",
  ],
  theme: {
    extend: {
      fontFamily: {
        sans: ['Nunito', 'sans-serif'],
      }
    },
  },
  plugins: [],
}
```

in `tailwind.config.js` it was perfect: `Nunito` was the default font.

Fast forward to today, I am with a simple app (again Vite/Vue3) I develop for some time and I decided to change the default font by exactly following the same two steps.

**My problem: the default font does not change.**

I tried to remove the cache, restart the application, use a browser I never used - nothing helps.

I added, for debugging purposes, to `App.vue` (the topmost app where the loading of the font happens)

```css
<style>
div {
    font-family: 'Nunito';
}
</style>
```

Here the font changed (on `<div>` at least but this is more or less how apps today are structured - so in practice this was the whole app). So the font is present as expected.

When inspecting an element, I see that the computed font is set on `<html>`

```css
font-family: ui-sans-serif, system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, "Noto Sans", sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji";
```

and it is likely that this is what is set by Tailwind's Preflight.

**Is there something I forgot?**

  [1]: https://tailwindcss.com/docs/font-family#customizing-the-default-font

**Comment by cannap** [comment]
use @apply font-sans;

**Comment by wongjn** [comment]
Consider checking

```js
export default {
  // …
  theme: {
    extend: {
      fontFamily: {
        sans: ['Nunito', 'sans-serif'],
      }
    },
  },
}
```
is set in your Tailwind configuration. Otherwise, consider providing a remotely-hosted git repository that reproduces the unexpected behavior.

**Comment by wsw70** [comment, marked as answer]
I needed to apply `class="font-sans"` to the highest `<div>` in order to actually have the font used in the application. 
Not really sure why this is needed.

**Reply by cannap** [reply]
make a repo 
/e can you put it on the body to have everywhere the same maybe   something overrides check also computed props in devtools
